### DISCUSSION TITLE
Using typeorm to prompt for errors, Transforming JavaScript decorators to the configured target environment ("es2019") is not supported yet

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#23685] [Questions]
### Environment

- Operating System: Windows_NT                                                                                                                                         
- Node Version:     v18.17.1                                                                                                                                           
- Nuxt Version:     3.7.4                                                                                                                                              
- CLI Version:      3.9.0                                                                                                                                              
- Nitro Version:    2.6.3                                                                                                                                              
- Package Manager:  yarn@1.22.19
- Builder:          -
- User Config:      devtools, hooks
- Runtime Modules:  -
- Build Modules:    -

### Reproduction

Transforming JavaScript decorators to the configured target environment ("es2019") is not supported yet

### Describe the bug

Transforming JavaScript decorators to the configured target environment ("es2019") is not supported yet
![image](https://github.com/nuxt/nuxt/assets/26525599/d04512af-0112-416d-bfd6-86929a849519)

**Comment by Wanchai** [comment]
It comes from Vite or Nitro as I'm using AnalogJs and have the same problem.
